cmd/github-mcp-server/main.go

```diff
@@ -95,15 +107,36 @@ func initLogger(outPath string) (*log.Logger, error) {
 	return logger, nil
 }
 
-func runStdioServer(readOnly bool, logger *log.Logger, logCommands bool, exportTranslations bool) error {
+type runConfig struct {
```

Comment:
I'd probably just call this `config` but other than that lgtm!

Reply:
going to leave it so you don't have to do another review